Monika After Story/game/zz_backgrounds.rpy

```diff
@@ -337,21 +340,40 @@ init 800 python:
                 The background we're changing to
 
             by_user:
-                If the user switched the background themselves
+                True if the user switched the background themselves
+
+            set_persistent:
+                True if we want this to be persistent
         """
         if by_user is not None:
             mas_background.force_background = bool(by_user)
 
         mas_current_background.exit(new_background)
         mas_setBackground(new_background)
 
+    def mas_startupBackground():
+        """
+        Sets up the spaceroom to start up in the background you left in if it is unlocked and still exists
+        """
+        if (
+            persistent._mas_current_background in store.mas_background.BACKGROUND_MAP
```

Comment:
what is the plan if `persistent._mas_current_background` is invalid? does it get reset to a valid value somewhere else?